Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -9791,19 +9791,13 @@ void WebPageProxy::setTextIndicatorFromFrame(FrameIdentifier frameID, WebCore::T
     if (!frame)
         return;
 
-    RefPtr rootFrameParent = frame->rootFrame().parentFrame();
-    if (!rootFrameParent) {
-        setTextIndicator(WTFMove(indicatorData), lifetime);
-        return;
-    }
```

Comment:
Removing these early return causes us to always send a useless async IPC message even when site isolation is disabled, was that intended?

Reply:
Oh, I see. I missed that the root frameID was being passed.